onnx/common/common.h

```diff
@@ -2,7 +2,30 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-
 #pragma once
 
 #define ONNX_UNUSED_PARAMETER(x) (void)(x)
+
+#ifdef ONNX_NO_EXCEPTIONS
+#define ONNX_THROW(...)                                 \
+  std::cerr << ONNX_NAMESPACE::MakeString(__VA_ARGS__); \
+  abort();
+
+#define ONNX_THROW_EX(ex)                \
+  do {                                   \
+    std::cerr << ex.what() << std::endl; \
+    abort();                             \
+  } while (false)
+
+#define ONNX_TRY if (true)
+#define ONNX_CATCH(x) else if (false)
+#define ONNX_HANDLE_EXCEPTION(func)
+
+#else
+#define ONNX_THROW(...) throw std::runtime_error(ONNX_NAMESPACE::MakeString(__VA_ARGS__));
```

Comment:
Drop semicolon?

Reply:
Nit: why do we need the semicolon in the macro? Seems preferable to use same style everywhere.